modules/ROOT/pages/tools/cypher-shell.adoc

```diff
@@ -196,6 +201,88 @@ Type :help for a list of available commands or :exit to exit the shell.
 Note that Cypher queries must end with a semicolon.
 ----
 
+[[cypher-shell-access-mode]]
+== Changing the access mode
+
+_This feature is available in Neo4j 5.20 and later._
+
+By default, the access mode is set to `write`.
+However, you can change the access mode to `read` or `write` using the `--access-mode` argument when connecting to a Neo4j DBMS with the `cypher-shell` command or by using the `:access-mode` command in the interactive shell.
```

Comment:
Should we mention that access mode can affect to which nodes in a cluster a query can get routed to (read replicas can only do reads). Or is that too much?

Reply:
I think it'll be good to add it.  Would that be correct to say:
_Keep in mind that access mode can affect which servers in a cluster a query can get routed to. For example, a server with `modeConstraint=SECONDARY` can only do reads._